docs/op-stack/src/docs/security/pause.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Pause and Unpause the Bridge
+lang: en-US
+---
+
+
+## Why do it?
+
+The bridge allows movement between blockchains. 
```

Comment:
"The `OptimismPortal` is a bridge contract that makes it possible to send messages between your L1 and your L2 OP Stack chain. The `OptimismPortal` is pausable as a backup safety mechanism that allows a specific `GUARDIAN` address to temporarily halt deposits and withdrawals to mitigate security issues if necessary. An OP Stack chain does not have to specify a `GUARDIAN` address if it does not want to make the `OptimismPortal` contract pausable.